src/Client/Web/Bit.Client.Web.BlazorUI/Components/ProgressIndicator/BitProgressIndicator.razor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitProgressIndicator
+    {
+        private bool PercentCompleteHasBeenSet;
+        private bool DescriptionHasBeenSet;
+        private double percentComplete;
+        private string description = string.Empty;
+        [Parameter] public string Label { get; set; } = string.Empty;
+
+        [Parameter]
+        public double PercentComplete
+        {
+            get => percentComplete;
+            set
+            {
+                if (value == percentComplete) return;
+                percentComplete = Normalize(value);
+                _ = PercentCompleteChanged.InvokeAsync(value);
+            }
+        }
+
+        [Parameter]
+        public string Description
```

Comment:
this parameter is not a 2-way bound param, please correct it and remove all related implementations